pkg/apis/servicecatalog/types.go

```diff
@@ -208,6 +208,9 @@ type InstanceSpec struct {
 // InstanceStatus represents the current status of an Instance.
 type InstanceStatus struct {
 	Conditions []InstanceCondition
+	// AsyncOpInProgress is set to true if there is an ongoing async operation
+	// against this Service Instance in progress.
+	AsyncOpInProgress bool
```

Comment:
Well, it's in the Spec currently (as OSBLastOperation or something like that) but I need to read and see if it should be moved here, I think it's the wrong place for it, but I haven't had time to delve into the details about that move. I'd rather do that in a follow on PR as this is getting large enough already, and I'd rather move this forward.